app/scripts/modules/amazon/common/footer.directive.js

```diff
@@ -3,15 +3,15 @@
 let angular = require('angular');
 
 module.exports = angular
-  .module('spinnaker.amazon.serverGroup.details.resize.footer.directive', [
+  .module('spinnaker.amazon.footer.directive', [
   ])
-  .directive('awsResizeFooter', function () {
+  .directive('awsFooter', function () {
```

Comment:
This should probably be "modalFooter", if we are going to abstract it away from the resize modal, but I will do that as part of a separate PR this afternoon.